compiler/lexer.nim

```diff
@@ -340,10 +340,15 @@ proc getNumber(L: var TLexer, result: var TToken) =
       else:
         break
       if buf[pos] == '_':
-        if buf[pos+1] notin chars:
+        if buf[pos+1] notin chars and buf[pos+1] == '_':
```

Comment:
This is the same as ``if buf[pos+1] == '_'`` and makes little sense.

Reply:
Heh, good point. Tired me coding ... :)